jme3-glsl-highlighter/src/com/jme3/gde/glsl/highlighter/lexer/GlslLanguageHierarchy.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.jme3.gde.glsl.highlighter.lexer;
+
+import java.util.Collection;
+import java.util.EnumSet;
+import org.netbeans.spi.lexer.LanguageHierarchy;
+import org.netbeans.spi.lexer.Lexer;
+import org.netbeans.spi.lexer.LexerRestartInfo;
+
+/**
```

Comment:
A description what a LanguageHierarchy does would be nice, but that's just a note for the future, not relevant for this pr

Reply:
Sadly I have no idea...